doc/index.rst

```diff
@@ -63,6 +63,13 @@ Reference documentation
 
    plan_stubs/*
 
+.. toctree::
+   :maxdepth: 2
+   :caption: Plans (Reflectometry)
+   :glob:
+
+   plans/reflectometry/*
```

Comment:
if #111 gets merged before it has the refl-specific plans just in plans.md, we should decide on a structure. 

maybe plans should be the toctree with a separate _file_ for technique specific stuff?

Reply:
I've made an initial pass at this, I think a more general look at all of the docs from our 3 big PRs will be good to do this when all 3 PRs are in.